### DISCUSSION TITLE
"Villager Access Trust"

### DISCUSSION THREAD

**Discussion by Uboaaaaa** [GriefPrevention/GriefPrevention#1329] [Ideas]
**As** a player and a server owner
**I want** an trust option that allows players access villagers.
**Because** I have attempted to build a public villager trading hall for the community. It is claimed to prevent griefing. However the public cannot have access to the villager trades unless they are given container trust. Which in turn would allow them to mess around with the containers within the same claim. It would be nice if there's a separate claim trust permission that listens for trading with villagers.

**Comment by RoboMWM** [comment]
I believe this feature exists in the config.

**Reply by Uboaaaaa** [reply]
Yes, there is **VillagerTradingRequiresPermission**, however not all players on the server wants others to trade with their villagers. Therefore I'd like to request a separate permission that handles villager trading solely.

**Comment by bigpresh** [comment]
Personally I think adding a new trust level for this would just make things more confusing.  My solution would be to place a subclaim covering the villagers but not the containers, and grant public containertrust in the subclaim.

Otherwise, I think it's something better handled by a plugin - the Flags plugin quite possibly already allows this?
